public/index.css

```diff
@@ -0,0 +1,52 @@
+* {
```

Comment:
This should be moved back to be live inside `src` and imported in `index.js` file. Generally what is inside the `public` directory should be the things you won't need to edit long term.

Reply:
I actually spent a bunch of time trying to figure out where they should go, so thanks! 

Most of the styling in there applies to the loading screen and the footer, which are rendered directly by `index.html`. If it's imported by `index.js` then they wouldn't be applied fast enough.

I figured the  `*, html` selectors could potentially go in a `reset.css`, but I wasn't sure if that would go in `public/` or `src/`. It seemed cleaner to just keep it together in `public/`, especially since it would affect the loading screen style too.